lib/clusterManager.js

```diff
@@ -120,6 +120,10 @@ if (cluster.isMaster) {
     var inits = [];
     inits.push(this.initSecret.bind(this));
     inits.push(this.initSecret.bind(this));
+    if (process.env.ZOWE_SESSION_SECRET) {
```

Comment:
is this actually secure, or can it be spied on? at least, i think we need to change zlux-app-server which prints all env vars at startup. It should omit printing things that say password or secret.

Reply:
I created a PR https://github.com/zowe/zlux-app-server/pull/120